src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeConverter.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    internal class AttributeCloner
+    {
+        // $$$ Merge this with other methods on this class. 
```

Comment:
It turns out this path it doesn't need to do any resolution.  Next iteration will be even simpler. 

---
In reply to: [108521724](https://github.com/Azure/azure-webjobs-sdk/pull/1093#discussion_r108521724) [](ancestors = 108521724)